custom/intrahealth/report_calcs.py

```diff
@@ -371,3 +371,66 @@ def total(self, form):
             'date': form.form['mois_visite'],
             'value': value
         }
+
+
+class Recouvrement(fluff.Calculator):
+
+    @fluff.date_emitter
+    def amount_to_pay(self, form):
```

Comment:
not a big deal but it looks like all of these could call a shared function with a different property name which would reduce copy/pasted code